tdrs-backend/tdpservice/security/management/commands/process_owasp_scan.py

```diff
@@ -0,0 +1,107 @@
+"""Management command to process OWASP scan results from CircleCI."""
+
+from tempfile import TemporaryFile
+import logging
+import requests
+
+from django.core.files import File
+from django.core.management import BaseCommand
+
+from tdpservice.security.models import OwaspZapScan
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """Command to process OWASP scan results."""
+
+    help = 'Download, process and store OWASP ZAP scan results.'
+
+    def add_arguments(self, parser):
+        """Specify accepted arguments for this command."""
+        parser.add_argument('build_num', type=int)
+        parser.add_argument(
+            '--backend-pass-count',
+            default=0,
+            type=int
+        )
+        parser.add_argument(
+            '--backend-warn-count',
+            default=0,
+            type=int
+        )
+        parser.add_argument(
+            '--backend-fail-count',
+            default=0,
+            type=int
+        )
+        parser.add_argument(
+            '--frontend-pass-count',
+            default=0,
+            type=int
+        )
+        parser.add_argument(
+            '--frontend-warn-count',
+            default=0,
+            type=int
+        )
+        parser.add_argument(
+            '--frontend-fail-count',
+            default=0,
+            type=int
+        )
+        parser.add_argument(
+            '--project-slug',
+            default='raft-tech/TANF-app',
+            type=str
+        )
+
+    def handle(self, *args, **options):
+        """Given a build number, retrieve the OWASP Zap scan reports."""
+        circle_build_num = options['build_num']
+        circle_project_slug = options['project_slug']
+        circle_api_url = (
+            f'https://circleci.com/api/v2/project/gh/{circle_project_slug}/'
+            f'{circle_build_num}/artifacts'
+        )
+        response = requests.get(circle_api_url)
+        artifacts = response.json().get('items', [])
+        for artifact in artifacts:
+            report_path = artifact.get('path')
+            report_url = artifact.get('url')
+
+            if not (report_path and report_url):
+                raise ValueError(
+                    f'Build {circle_build_num} has no valid artifacts for '
+                    f'project {circle_project_slug}'
+                )
+
+            app_target = report_path.split('/')[0]
+
+            # Determine the matching alert metrics keys
```

Comment:
This extracts the relevant variables from the `options` dictionary and sets them to 0 if they are not found.